<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame
{
	background-color: rgb(156, 156, 156);
     border-radius:12px;     
}
QLabel
{
color:white;
}
QLineEdit{
border-radius:6px;     
}
QPushButton{
	color: rgb(255, 255, 255);
border-radius:6px;     
	background-color: rgb(172, 217, 255);
}
QPushButton:pressed
{
	background-color: rgb(240, 213, 255);
}
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>361</width>
     <height>271</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hiragino Sans GB</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>用户名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtUserName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hiragino Sans GB</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>密 码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPassWord">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>STSong</family>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hiragino Sans GB</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>登 录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegister">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hiragino Sans GB</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>注 册</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
